app/models/patient.rb

```diff
@@ -81,4 +106,17 @@ def call_result=(new_call_result)
 
     super(new_call_result)
   end
+
+  def self.not_contacted
```

Comment:
This can become a named scope.

Reply:
I'm resolving this since it's unrelated to this PR.